### DISCUSSION TITLE
Make scratchpad bigger on multiple monitors with different aspect ratios

### DISCUSSION THREAD

**Discussion by smitropoulos** [i3/i3#5815] [Q&A]
Hello! 

I like how my scratchpad gets resized on my vertical monitor. My main monitor is 2560x1440 and my second is 1440x2560. 

When I open my scratchpad in either orientation, it is centered and has a certain width and height which gets adjusted automatically!

I want the scratchpad to be bigger, but still adjust to borders of the display. 

So I would like it to be centered, and 80% of both height and width, no matter the orientation of the display. 

Is that something that can be done? I have so far tried ppt resizing and px resizing which ofc will not work since main monitor is 16:9 and the vertical is 9:16

**Comment by orestisfl** [comment]
Something like `resize set 80 ppt 80 ppt` should resize a floating window to 80% of the screen, does it work for you?

**Comment by smitropoulos** [comment]
This resize I reckon gets the dimensions of my Xscreen, which is 6560x2560 pixels (1735x677 millimeters) (taken from nvidia-settings) and thus make the window a percentage of this.

![image](https://github.com/i3/i3/assets/21141766/b000f661-9037-4a44-9c32-428cae03cf89)

This means that the floating window gets extremely big, 80% of screen instead of monitor size

This is all a guess btw.

**Reply by orestisfl** [reply]
Ah nice, an nvidia edge case. Please open a bug report: https://github.com/i3/i3/issues/new?assignees=&labels=&projects=&template=bug_report.md

If you have time to do that, you just need to focus on the `resize set <x> ppt` since that's the thing that's broken. A log and i3 version will be helpful here.
